definitions/json/ComplexTypes.json

```diff
@@ -5412,6 +5483,14 @@
     "ElementLabel": "Status",
     "SecurityClassification": "Public"
   },
+  {
+    "ID": "referralDetails",
+    "ListElementCode": "referralReplyCollection",
+    "FieldType": "Collection",
+    "ElementLabel": "Reply",
+    "FieldTypeParameter": "referralReply",
+    "SecurityClassification": "Public"
+  },
   {
     "ID": "CreateReferralDocumentUpload",
```

Comment:
I know this isnt related to this PR but same as the other comment, why is this its own ComplexType and not just using DocumentUpload?

Reply:
I will change it for both of the Create and Reply referral event to use ReferralDocumentUpload type, the complex type can be shared.

Reply:
why do we need a new type for this? We can re-use DocumentUpload and just disable that extra field in EventToComplexTypes

Reply:
Yep I have made the changes as suggested by Harpreet.